tests/unit/base/SGObject_unittest.cc

```diff
@@ -311,3 +314,73 @@ TEST(SGObject,parameter_hash_changed)
 	SG_UNREF(inf);
 }
 #endif //USE_GPL_SHOGUN
+
+TEST(SGObject, tags_set_get_string_sgvector)
+{
+	auto obj = some<CMockObject>();
```

Comment:
trais is ok
precise buildbot will, see [here](http://buildbot.shogun-toolbox.org/builders/precise%20-%20libshogun)

@vigsterkr I guess precise can go offline as of this merge now